common.yaml

```diff
@@ -165,6 +167,43 @@ postprocess:
             done
         done
     done
+  - |
+     #!/usr/bin/env bash
+     set -xeo pipefail
+
+     # FIXME: Why is this only broken here?  NM isn't removing the link?
+     sed -i '/etc.resolv/d' /usr/lib/tmpfiles.d/etc.conf
+
+     # crio should stop hardcoding things in their config file!
+     # We are apparently somehow pulling in a conmon override in RHCOS
+     # that contains /usr/libexec/crio/conmon - WHY?
+     # sed -i '/conmon.*=/d' /etc/crio/crio.conf
+     # Oh right but the MCO overrides that too so...
+     mkdir -p /usr/libexec/crio
+     ln -sr /usr/bin/conmon /usr/libexec/crio/conmon
+
+     # Enable tmp-on-tmpfs by default because we don't want to have things leak
+     # across reboots, it increases alignment with FCOS, and also fixes the
+     # Live ISO. First, verify that RHEL is still disabling.
+     grep -q '# RHEL-only: Disable /tmp on tmpfs' /usr/lib/systemd/system/basic.target
+     echo '# RHCOS-only: we follow the Fedora/upstream default' >> /usr/lib/systemd/system/basic.target
+     echo 'Wants=tmp.mount' >> /usr/lib/systemd/system/basic.target
+  - |
+     #!/usr/bin/env bash
+     set -xeo pipefail
+     # See https://issues.redhat.com/browse/LOG-3117
+     # Something changed between rhel8 and rhel9 to not generate this by default
+     # but we have containers that expect it to be mounted so for now let's continue
+     # generating it.
+     ln -sr /usr/share/zoneinfo/UTC /etc/localtime
+  - |
+    #!/usr/bin/env bash
+    set -xeuo pipefail
+    # Backport https://github.com/dracutdevs/dracut/commit/25a92885a9519701cc480298c2b082e2e2bf5ebe
+    s=/usr/lib/dracut/modules.d/95nvmf/nvmf-autoconnect.sh
+    if test -f "$s"; then
+      chmod a+x "$s"
+    fi
```

Comment:
We can remove workaround for `nvmf-autoconnect.sh`, as the issue does not exist on 9.2
```
[cloud-user@citest-1 ~]$ rpm -q dracut
dracut-057-21.git20230214.el9.x86_64
[cloud-user@citest-1 ~]$ ll /usr/lib/dracut/modules.d/95nvmf/nvmf-autoconnect.sh
-rwxr-xr-x. 1 root root 140 Jun 19  2022 /usr/lib/dracut/modules.d/95nvmf/nvmf-autoconnect.sh
[cloud-user@citest-1 ~]$ cat /etc/os-release 
...
REDHAT_SUPPORT_PRODUCT_VERSION="9.2 Beta"
```

Reply:
Hmm yes...though this is in the FCOS config, so fixing that would require a cycle there...let's just keep this on our list of things to clean up later.